de-lib/src/main/java/org/iplantc/de/theme/base/client/fileViewers/PathListViewerDefaultAppearance.java

```diff
@@ -35,8 +37,12 @@ public String columnHeaderText() {
     }
 
     @Override
-    public String pathListViewName(String name) {
-        return fileViewerStrings.pathListViewName(name);
+    public String pathListViewName(String infoType, String name) {
+        if (infoType.equals(InfoType.HT_ANALYSIS_PATH_LIST.getTypeString())) {
```

Comment:
We should probably flip this check as well, just in case `infoType` is null: `InfoType.HT_ANALYSIS_PATH_LIST.getTypeString().equals(infoType)`